Supress not used test & benchmark functions

_test.go files have Benchmark... and Test... functions - it isn't use in main program and linters warning many times about dead code.

I think these errors have to be supress by default for these functions.

Minimal example:

bug repeated for file test_test.go

```golang
package main

import "testing"

func TestTest(t *testing.T) {

}
```

```bash
golangci-lint --version
golangci-lint has version 1.7.2 built from 93311ae on 2018-06-19T13:43:26Z

golangci-lint run
test_test.go:5:6: `TestTest` is unused (deadcode)
func TestTest(t *testing.T) {
     ^
```

--- a/Gopkg.lock
+++ b/Gopkg.lock
@@ -69,7 +69,7 @@
   branch = "master"
   name = "github.com/golangci/go-misc"
   packages = ["deadcode"]
-  revision = "a61b005a223b50d3656798639dd87073ca5c1981"
+  revision = "927a3d87b613e9f6f0fb7ef8bb8de8b83c30a5a2"
 
 [[projects]]
   branch = "master"

--- a/test/testdata/deadcode_main_pkg/main_test.go
+++ b/test/testdata/deadcode_main_pkg/main_test.go
@@ -1,0 +1,15 @@
+package main
+
+import "testing"
+
+func TestF(t *testing.T) {
+
+}
+
+func BenchmarkF(t *testing.B) {
+
+}
+
+func ExampleF() {
+
+}

